tools/state-viewer/src/congestion_control.rs

```diff
@@ -189,6 +192,8 @@ impl PrepareBenchmarkCmd {
 
         for _ in 0..self.receipt_count {
             let receipt = self.create_receipt();
+            let receipt = Cow::Borrowed(&receipt);
+            let receipt = ReceiptOrStateStoredReceipt::Receipt(receipt);
             queue.push(&mut trie_update, &receipt).unwrap();
         }
 
```

Comment:
Rebasing on top of `master` makes it harder for the reviewer to figure out what changed between the reviewed versions, it kinda forces me to read the whole change again. I think merging master into the PR branch is better, github shows a nice "compare" button and it filters out all the changes unrelated to this PR.

Reply:
Ah sorry, I'm more used to the rebase model. You're right, in git merge is preferred, I'll use that in the future.

Reply:
IMO rebasing makes more sense conceptually, but sadly github doesn't have the ability to show what has changed in the PR between the rebases :/ I tried to run git diff on the previous branch and the new one, but it included all of the new changes from master :/

Reply:
If it makes it any easier I kept the same commit history so the only discrepancy in older commits would be the resolved merge conflicts.